dipy/reconst/csdeconv.py

```diff
@@ -139,7 +140,8 @@ def __init__(self, gtab, response, reg_sphere=None, sh_order=8, lambda_=1,
         else:
             self.sphere = reg_sphere
 
-        r, theta, phi = cart2sphere(self.sphere.x, self.sphere.y, self.sphere.z)
+        r, theta, phi = cart2sphere(
+            self.sphere.x, self.sphere.y, self.sphere.z)
```

Comment:
I prefer the breakdown you did on line 294 below, with each one of these on its own line.